assets/src/components/mapPage/mapDisplay.tsx

```diff
@@ -212,7 +212,7 @@ const RoutePatternLayers = ({
 }: {
   routePattern: RoutePattern
   isSelected: boolean
-  onShapeClick: () => void
+  onShapeClick?: () => void
```

Comment:
**suggestion:** with these changes, am I correct that `onShapeClick` will no longer be used for anything? If so, it may make the code a bit cleaner if it's removed, along with downstream places it's passed, mainly `onClick` for `<RouteShape/>` assuming that's not used for anything else.

Reply:
a7a23b828b0f8cf860cc8b5075cd9e05d22bd8d9